cmd/sharded-test-server/main.go

```diff
@@ -223,14 +223,14 @@ func start(proxyFlags, shardFlags []string, logDirPath, workDirPath string, numb
 	// start shards
 	var shards []*shard.Shard
 	for i := 0; i < numberOfShards; i++ {
-		shard, err := newShard(ctx, i, shardFlags, standaloneVW, servingCA, hostIP.String(), logDirPath, workDirPath, cacheServerConfigPath, clientCA)
```

Comment:
IMO better not to alias the test server package as `shard`.

Reply:
changed